SignalServiceKit/src/Messages/Interactions/OWSContact.h

```diff
@@ -102,6 +102,8 @@ typedef NS_ENUM(NSUInteger, OWSContactAddressType) {
 
 - (instancetype)init NS_UNAVAILABLE;
 
+- (void)normalize;
```

Comment:
I'm thinking we should be explicit about normalizing the data, e.g. do it explicitly during the "share contact" workflow.